### PR TITLE
Security/TLS: DTLS 1.2 image update & corrections

### LINKED ISSUE #4031: DTLS 1.1
Since there is not DTLS 1.1 based on this page. "DTLS 1.0 is based on TLS 1.1, and DTLS 1.2 is based on TLS 1.2. There is no DTLS 1.1; that version number was skipped in order to harmonize version numbers with TLS" https://en.wikipedia.org/wiki/Datagram_Transport_Layer_Security

Is the DTLS 1.1 reg key display above correct or should it be DTLS1.2?

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: dd180142-4c19-8abc-ecce-e4c7e5ac8951
* Version Independent ID: fc5bba87-afad-1729-8f7e-333c73188b73
* Content: [Transport Layer Security (TLS) registry settings](https://docs.microsoft.com/en-us/windows-server/security/tls/tls-registry-settings)
* Content Source: [WindowsServerDocs/security/tls/tls-registry-settings.md](https://github.com/MicrosoftDocs/windowsserverdocs/blob/master/WindowsServerDocs/security/tls/tls-registry-settings.md)
* Product: **windows-server**
* Technology: **security-tls-ssl**
* GitHub Login: @Justinha
* Microsoft Alias: **justinha**

### LINKED ISSUE #4627: DTLS 1.1 (not a thing)
Hi,
under the DTLS 1.2 heading, it refers to "DTLS 1.1"
Reasonably sure DTLS 1.1 isn't a thing. Should be 1.0 1.2 or 1.3

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: dd180142-4c19-8abc-ecce-e4c7e5ac8951
* Version Independent ID: fc5bba87-afad-1729-8f7e-333c73188b73
* Content: [Transport Layer Security (TLS) registry settings](https://docs.microsoft.com/en-us/windows-server/security/tls/tls-registry-settings)
* Content Source: [WindowsServerDocs/security/tls/tls-registry-settings.md](https://github.com/MicrosoftDocs/windowsserverdocs/blob/master/WindowsServerDocs/security/tls/tls-registry-settings.md)
* Product: **windows-server**
* Technology: **security-tls-ssl**
* GitHub Login: @Justinha
* Microsoft Alias: **justinha**

### LINKED ISSUE #4731: Invalid DTLS 1.1 references
DTLS 1.2 version section (https://docs.microsoft.com/en-us/windows-server/security/tls/tls-registry-settings#dtls-12) contains references to DTLS 1.1 in the example - both text and the screenshot show DTLS 1.1

There is no such thing as DTLS 1.1.
Please correct the example text and the screenshot to show DTLS 1.2

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: dd180142-4c19-8abc-ecce-e4c7e5ac8951
* Version Independent ID: fc5bba87-afad-1729-8f7e-333c73188b73
* Content: [Transport Layer Security (TLS) registry settings](https://docs.microsoft.com/en-us/windows-server/security/tls/tls-registry-settings)
* Content Source: [WindowsServerDocs/security/tls/tls-registry-settings.md](https://github.com/MicrosoftDocs/windowsserverdocs/blob/master/WindowsServerDocs/security/tls/tls-registry-settings.md)
* Product: **windows-server**
* Technology: **security**
* GitHub Login: @Justinha
* Microsoft Alias: **justinha**

### LINKED ISSUE #4976: Incorrect column heading
Under the section "SendTrustedIssuerList" there is a table with the title "Default Send Trusted Issuer List behavior".  The second column currently has a header of "Time", which does not apply to the values listed (i.e. TRUE, FALSE).  It's more likely this header should be something like "Default value", or "Default Behavior".

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: dd180142-4c19-8abc-ecce-e4c7e5ac8951
* Version Independent ID: fc5bba87-afad-1729-8f7e-333c73188b73
* Content: [Transport Layer Security (TLS) registry settings](https://docs.microsoft.com/en-us/windows-server/security/tls/tls-registry-settings)
* Content Source: [WindowsServerDocs/security/tls/tls-registry-settings.md](https://github.com/MicrosoftDocs/windowsserverdocs/blob/master/WindowsServerDocs/security/tls/tls-registry-settings.md)
* Product: **windows-server**
* Technology: **security**
* GitHub Login: @Justinha
* Microsoft Alias: **justinha**

### PR DESCRIPTION
**Pull request by KarkDenUheldige**
**Description:**

As reported in issue tickets #4031 (**DTLS 1.1**), #4627 (**DTLS 1.1 (not a thing)**), #4731 (**Invalid DTLS 1.1 references**), and #4976 (**Incorrect column heading**), there are several details in need of update in this page. Among them is the need to replace the DTLS 1.1 fictional information with factual DTLS 1.2 information.

1. The example for "DTLS 1.1" mentions a non-existing protocol
2. Column header "Time" over the values TRUE/FALSE (boolean)
3. Typo: "explcitly" (instead of explicitly)

**Changes proposed:**
- Replace the existing DTLS 1.1 image with a DTLS 1.2 image
- Replace text containing DTLS 1.1 with DTLS 1.2 .
- Replace the column header "Time" with "Default behavior"
- Replace "explcitly" with 'explicitly'

**Additional whitespace changes:**

- add missing MD indent marker compatibility spacing in the "Applies to:" line
- add centering in table column 2 above TRUE T& FALSE
- remove all underscores from line 218 (H2 codestyle)
- add NewLine to the end of the document page

**Post Scriptum:**

Thanks to @RAJU2529 for his patience and for inspiring me to create this PR, seeing that I was unable to contribute directly to his existing PRs .

**Ticket closure or reference(s):**

Closes #4031
Closes #4627
Closes #4632
Closes #4731
Closes #4737
Closes #4976

### CODE REVIEW

**Comment by PRMerger20** [comment]
@illfated : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Review by illfated — COMMENTED** [review]
Default source file view: https://github.com/MicrosoftDocs/windowsserverdocs/pull/4988/files

image preview (only the new image) (rich diff): https://github.com/MicrosoftDocs/windowsserverdocs/pull/4988/files?short_path=a63a2aa#diff-a63a2aae55bd2a9af8ca88f6838ce4cf470cde669e7fa7aa543df7e5f0e84249

rich diff view for the document text: https://github.com/MicrosoftDocs/windowsserverdocs/pull/4988/files?short_path=95524aa#diff-95524aa8d8700d68adb9622e8536e06d3bf5bf4467bd69e1727f05597555d9f8

**Comment by illfated** [comment]
#sign-off

**Comment by PRMerger13** [comment]
@illfated: I'm sorry - only the author of this article, @justinha, can sign off on your changes. But we do have an exception process - if you are on the Microsoft content or product team for this product area, you can ask the PR review team to review and merge it by sending mail to the techdocprs alias.

**Comment by ixtlanian** [comment]
Thanks for the proposed updates. One additional thought to Microsoft Documentation team:

- Please fix any other DTLS 1.1 references across the entire MIcrosoft Docs ecosystem.
Here is [an example of another document referencing a non-existent DTLS 1.1](https://docs.microsoft.com/en-us/azure/rtos/netx-duo/netx-secure-dtls/chapter3). That one seems to be in the location where I cannot submit any feedback.

Feel free to separate those into additional issue.

**Comment by illfated** [comment]
@ixtlanian : Thank you for the useful feedback. I am prepared to create additional Pull Requests to have the other pages corrected.

By the way, I have answered a handful of tickets where other users have claimed that the images do not show the correct state of the services being disabled, because they (in my opinion) claim that the Registry key **`DisabledByDefault`** should have the value 1 instead of 0, whereas I find that the images seem to be correct by showing the key **`Enabled = 0`**. Do you have any views on that detail?

**Comment by ixtlanian** [comment]
> @ixtlanian : Thank you for the useful feedback. I am prepared to create additional Pull Requests to have the other pages corrected.
> 
> By the way, I have answered a handful of tickets where other users have claimed that the images do not show the correct state of the services being disabled, because they (in my opinion) claim that the Registry key **`DisabledByDefault`** should have the value 1 instead of 0, whereas I find that the images seem to be correct by showing the key **`Enabled = 0`**. Do you have any views on that detail?

Well, this is a standard case where Microsoft makes things difficult with double negation and further complicates it with dual flags overriding each other. So let's break it down.

- DisabledByDefault and Enabled control different things, but we can think of DisabledByDefault as a subset of Enabled=1 setting.
- When Enabled flag is set to 0, the respective protocol version is disabled and cannot be negotiated no matter what the value of DisabledByDefault flag is. In this cases the value of DisabledByDefault is essentially ignored.
- If Enabled=1 AND DisabledByDefault=1 the respective protocol is not advertised, but it can still be explicitly requested through SSPI.
- If Enabled=1 AND DisabledByDefault=0 the respective protocol is available and advertised.
- In short, DisabledByDefault flag is somewhat useless and confusing because no matter what the value is here, it does not really disable the protocol.

So, back to your question, while the clean settings for a completely disabled protocol are:
- DisabledByDefault=1 (Disabled by Default)
- Enabled=0 (Disabled)

in reality, the following combination also completely disables the protocol:
- DisabledByDefault=0 (Enabled by Default)
- Enabled=0 (Disabled)

**Comment by illfated** [comment]
Thank you very much. It is always great to see someone as knowledgeable as you on a certain topic answering a question with all necessary details included. Very good, sir.

**Comment by illfated** [comment]
Reminder to myself: follow up on ticket #4926 (or open a new one) to add clarification about the 2 Registry keys and their real-world actual functionality, to reduce the amount of confusion in this topic detail.

**Comment by illfated** [comment]
Also ref. my closed PR #4987, which did not work as intended, causing me to create my own PR (this one).

**Comment by illfated** [comment]
#sign-off

**Comment by PRMerger18** [comment]
@illfated: I'm sorry - only the author of this article, @justinha, can sign off on your changes. But we do have an exception process - if you are on the Microsoft content or product team for this product area, you can ask the PR review team to review and merge it by sending mail to the techdocprs alias.

**Comment by PRMerger7** [comment]
@illfated : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by illfated** [comment]
#sign-off

**Comment by PRMerger7** [comment]
@illfated: I'm sorry - only the author of this article, @justinha, can sign off on your changes. But we do have an exception process - if you are on the Microsoft content or product team for this product area, you can ask the PR review team to review and merge it by sending mail to the techdocprs alias.

**Comment by illfated** [comment]
@Kiwifulla : Could we continue our discussion in this PR?

**Comment by PRMerger14** [comment]
@illfated : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by illfated** [comment]
Sorry, it was a difficult day yesterday. I wasn't paying attention enough and made some mistakes in my branch updates.

**Comment by illfated** [comment]
#sign-off

**Comment by PRMerger6** [comment]
@illfated: I'm sorry - only the author of this article, @justinha, can sign off on your changes. But we do have an exception process - if you are on the Microsoft content or product team for this product area, you can ask the PR review team to review and merge it by sending mail to the techdocprs alias.

**Comment by illfated** [comment]
Yes, I know, but this is currently the only way we can signal that the PR is ready for final review & merge.
